bot.js

```diff
@@ -1,140 +1,72 @@
+'use strict';
+
 var config = require('./config.js');
-var monkey_island = require('./monkey_island.js');
+var oneLiners = require('./one_liners.js');
+var replies = require('./replies.js');
 
 var TelegramBot = require('node-telegram-bot-api');
 
 var bot = new TelegramBot(config.token, {polling: true});
 
 /**
- * Returns an insult
+ * Replies with an insult
+ *
  * @param {object} message - The message to reply to
  * @param {object} user - The user to insult
- * @returns {string} The insult
  */
 function replyInsult(message, user) {
   if (Math.random() * 20 > 1) {
```

Comment:
And now it's all clear why it insults so often ^^ Random * 20 leads to evenly distributed values [0..20] which are > 1 in 19 of 20 cases which is ~95%